parler/models.py

```diff
@@ -786,7 +867,20 @@ def safe_translation_getter(self, field, default=None, language_code=None, any_l
             return default
 
 
-class TranslatableModel(TranslatableModelMixin, models.Model):
+class TranslatableModelBase(ModelBase):
+    """
+    Meta-class for model with translations.
+
+    """
+    def __new__(mcs, name, bases, attrs):
+        new_model = super(TranslatableModelBase, mcs).__new__(mcs, name, bases, attrs)
+        # new_model already has filled _meta structure here, it needs to access to model's pk
+        if not new_model._meta.abstract and not new_model._meta.proxy:
+            new_model.create_translations_related_virtual_fields()
+        return new_model
+
+
+class TranslatableModel(six.with_metaclass(TranslatableModelBase, TranslatableModelMixin, models.Model)):
```

Comment:
Note to self: It would be good to point out in the CHANGES that `TranslatableModel` uses a meta class now. Projects that define a meta class for any inherited class need to take note of this.

Reply:
Yep, there are no way to escape to do meta-class for TranslatableModel if we want to support django model inheritance. And we want - we have test_inherited_model in tests.